src/main/resources/edu/arizona/sista/reach/biogrammar/events_master.yml

```diff
@@ -106,8 +118,113 @@ rules:
       label: "Methylation"
       verbalTriggerLemma: "methylate"
       nominalTriggerLemma: "methylation"
+      triggerPrefix: "!word=/(?i)^de/"
+      priority: "5"
+
+
+  #
+  # "Reverse" Simple Events:
+
+  # Dephosphorylations
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dephosphorylation"
+      actionFlow: "mkBioMention"
+      label: "Dephosphorylation"
+      verbalTriggerLemma: "de-?phosphorylate"
+      nominalTriggerLemma: "de-?phosphorylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Deubiquitination
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Deubiquitination"
+      actionFlow: "mkUbiquitination"
+      label: "Deubiquitination"
+      verbalTriggerLemma: "de-?ubiquitinate"
+      nominalTriggerLemma: "de-?ubiquitination"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Dehydroxylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dehydroxylation"
+      actionFlow: "mkBioMention"
+      label: "Dehydroxylation"
+      verbalTriggerLemma: "de-?hydroxylate"
+      nominalTriggerLemma: "de-?hydroxylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Desumoylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Desumoylation"
+      actionFlow: "mkBioMention"
+      label: "Desumoylation"
+      verbalTriggerLemma: "de-?sumoylate"
```

Comment:
Note that the trigger prefix, as per your suggestion, is the matching expression string "word=/(?i)^de/" and not the actual prefix string itself. Using the actual prefix string alone as a parameter is not possible since the matching *rule* needs to be modified, not the pattern it is matching.